ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -394,3 +394,33 @@ def test_tensorflow_eye(
         batch_shape=batch_shape,
         dtype=dtype,
     )
+
+    
+# band_part
+@handle_cmd_line_args
+@given(
+    dtype_and_input=_get_dtype_and_matrix(),
```

Comment:
You need to generate data for all parameters in the function, i.e. `input`, `num_lower`, `num_upper`, `name` in the `given` block and then pass them to the `test_frontend_function`